packages/@react-stately/radio/src/useRadioGroupState.ts

```diff
@@ -75,7 +75,7 @@ export function useRadioGroupState(props: RadioGroupProps): RadioGroupState  {
 
   return {
     name,
-    selectedValue,
+    selectedValue: selectedValue || null,
```

Comment:
I think it'd be more correct to add `undefined` to https://github.com/adobe/react-spectrum/pull/5193/files#diff-04c4020464148b385bc97541606e76104e6a4c8e2d9edbbeb322a137ab9d6681R45

technically a users could omit `value` or `defaultValue` which would produce this case